notes/notes.11tydata.js

```diff
@@ -1,5 +1,8 @@
 const {titleCase} = require("title-case");
 
+// This regex finds all hashtags in a string
+const hashtagRegExp = /#(\w+)/g
```

Comment:
This regex should match dashes, like above.
```suggestion
const hashtagRegExp = /#[\w-]+/g
```

Reply:
Good catch. I agree.